docs/unique-design-challenges.md

```diff
@@ -76,4 +77,25 @@ While not part of the core nostr protocol, zaps are popular enough that we shoul
 
 In this guide we will explore how current clients are explaining zaps (if they are) and where / how we can improve the overall user experience by doing a better job at arming the user with the necessary information.
 
-[Explore Zaps](https://nostrdesign.org/docs/how-to/zaps/)
\ No newline at end of file
+[Explore Zaps](https://nostrdesign.org/docs/how-to/zaps/)
+
+## Challenge #5: NIPs
+
+Unlike email clients (e.g. yahoo email client, microsoft outlook email client, gmail email client etc.) which (explaining in an oversimplified way) either support or don't support the email protocol, nostr apps have tens of NIPs, and potentially hundreds of NIP candidates as nostr continues to grow. 
```

Comment:
I would uppercase the names and avoid the "email client" ripetition:
_(e.g. Yahoo, Microsoft Outlook, Gmail email clients)_